assets/pages/new-contributions-landing/components/ContributionAmount.jsx

```diff
@@ -4,30 +4,40 @@
 
 import React from 'react';
 import { connect } from 'react-redux';
+import type { Dispatch } from 'redux';
 
 import { amounts, type Amount, type Contrib } from 'helpers/contributions';
 import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
-import { type IsoCurrency, type Currency, type SpokenCurrency, currencies, spokenCurrencies, detect } from 'helpers/internationalisation/currency';
+import { type IsoCurrency, type Currency, type SpokenCurrency, currencies, spokenCurrencies } from 'helpers/internationalisation/currency';
 import { classNameWithModifiers } from 'helpers/utilities';
 
 import SvgDollar from 'components/svgs/dollar';
 
-import { type State } from '../contributionsLandingReducer';
+import { type Action, selectAmount, selectOtherAmount } from '../contributionsLandingActions';
 
 // ----- Types ----- //
 
+/* eslint-disable react/no-unused-prop-types */
```

Comment:
There are quite a few of these exceptions around the codebase. @Ap0c what do you think about just turning off this linting rule?

Reply:
I think this only applies to a small proportion of cases where ESLint can't figure out where the props are being passed through to. My preference would be to keep the rule because it isn't causing a problem for most `PropTypes` declarations.

As a side note, I've found that some of the places we're using this comment we don't actually need to; I've removed them in #916.